tools/ci/build.ps1

```diff
@@ -5,6 +5,6 @@ if(!(Test-Path -Path "C:/byond")){
 }
 
 bash tools/ci/install_node.sh
-bash tools/build/build
+bash tools/build/build --error
```

Comment:
Why would you do this only for a Windows build?

Reply:
I think it's useful to only have one job that fails if there are warnings. For example if the only problem is that there are Dream Maker warnings the map and integration jobs will pass. I thought it would be more useful this way. I included an example in original post how it would look like.